### DISCUSSION TITLE
API Version 2.x issues

### DISCUSSION THREAD

**Discussion by IddoxLatifi** [scripthookvdotnet/scripthookvdotnet#1512] [Q&A] [answered]
I have ScriptHookV, LemonUI, the SHVDN3 package and the latest _**Nightly version 3.6.0 135.**_ 

With the normal version of ScriptHookVDotNet i can't open a terminal using F4, it crashes the game.

When I start the Nightly version, I get the message in the attached ![Unbenannt](https://github.com/user-attachments/assets/0ad06fce-7418-4295-864a-61b6fb88e87f) 

I Changing NativeUI.dll to NativeUI.3.dll didn't help either. I would be very grateful if someone could help me with this

**If I just insert a script, it works. But as soon as NativeUI is added, I get this error message. Here are the logs again**

https://pastebin.com/VPKbK0Nc

**Comment by kagikn** [comment]
The content of the deprecated message is kind of confusing, but it is **NOT** an error one but just a warning one. It tells you there's scripts that *may* stop working in the future but doesn't tell about how they're currently running, where they're actually running as intended.

**Reply by IddoxLatifi** [reply]
> The content of the deprecated message is kind of confusing, but it is **NOT** an error one but just a warning one. It tells you there's scripts that _may_ stop working in the future but doesn't tell about how they're currently running, where they're actually running as intended.

Thanks for the quick reply! I understood that too, but is there any possibility, e.g. a downgrade from GTAV with the corresponding version of ScriptHookVDotNet which runs all mods? I'm really a total noob when it comes to this sort of thing, I only want to play 3 scripts, I basically don't care if it's the latest version.

And what means this Part in the README.txt? 

> `**For Users**: If you have raw scripts without an API version notation by file name, you will need to specify in nightly versions. You can specify an API version by adding a dot and a version number right before the extension name (e.g. `Script.cs` to `Script.2.cs`).`

I thought there was a way to simply rename the scripts so that multiple scripts could be loaded. Then I'll probably try other scripts. Anyway, thank you very much for the answer!

**Reply by kagikn** [reply, marked as answer]
For the following paragraph;
> **For Users**: If you have raw scripts without an API version notation by file name, you will need to specify in nightly versions. You can specify an API version by adding a dot and a version number right before the extension name (e.g. Script.cs to Script.2.cs).

"raw scripts" in the first sentence means scripts that has the file extension `.cs` or `.vb`, not a DLL file. If you can't see any file extension such as `.txt` or `.dll`, you would want to disable to hide certain extension names in Explorer. In your case, you have 3 scripts that are already built into DLL assembly files and renaming them just disables them to use.

**Reply by IddoxLatifi** [reply]
> For the following paragraph;
> 
> > **For Users**: If you have raw scripts without an API version notation by file name, you will need to specify in nightly versions. You can specify an API version by adding a dot and a version number right before the extension name (e.g. Script.cs to Script.2.cs).
> 
> "raw scripts" in the first sentence means scripts that has the file extension `.cs` or `.vb`, not a DLL file. If you can't see any file extension such as `.txt` or `.dll`, you would want to disable to hide certain extension names in Explorer. In your case, you have 3 scripts that are already built into DLL assembly files and renaming them just disables them to use.

Thank you very much. That answere all my questions i had 👍 Still installing other Scripts now, maybe some will work!
